features/steps/simple.py

```diff
@@ -0,0 +1,54 @@
+from behave import *
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+
+# More about scenario outline 👀 here https://jenisys.github.io/behave.example/tutorials/tutorial04.html
+
+@given("I have access to the internet")
+def step_impl(context):
+    context.driver = webdriver.Chrome()
+    """
+    :type context: behave.runner.Context
+    """
+    context.driver.get("https://www.google.com")
+    assert "Google" in context.driver.title
+
+
+@step('I have entered {nickname} in google search')
+def step_impl(context, nickname):
+    """
+    :type context: behave.runner.Context
+    """
+    assert nickname != ''
+    elem = context.driver.find_element_by_name("q")
+    elem.clear()
+    elem.send_keys(nickname)
+    elem.send_keys(Keys.RETURN)
+
+
+@step('page contains {realName}')
+def step_impl(context, realName):
+    """
+    :type context: behave.runner.Context
+    """
+    assert realName in context.driver.page_source
+
+
+@when("I click on first github link")
+def step_impl(context):
+    """
+    :type context: behave.runner.Context
+    """
+    elem = context.driver.find_element_by_css_selector(
+        'a[href^="https://github.com"]')
+    elem.click()
+
+
+@then('it would contain {learningSkills}')
+def step_impl(context, learningSkills):
+    """
+    :type context: behave.runner.Context
+    """
+    assert learningSkills in context.driver.page_source
+    context.driver.close()
```

Comment:
steps shouldn't contain actions that don't correspond to their description
use hooks for performing pre- and post- conditions for the test (https://stackoverflow.com/questions/66015535/is-it-possible-to-specify-a-specific-after-feature-hook-for-each-behave-featur)

Reply:
Thanks, I fix it in commits:
- [move webdriver part frm tests to environment.py](https://github.com/d9nchik/selenium/pull/1/commits/6c11b85f6fbca4f70fc7dd0e0c82f9bb0cdd1a98)
- [remove chromedriver path from environment.py](https://github.com/d9nchik/selenium/pull/1/commits/50ec3d1e2deeb468dc9b89d66df137b38b9fe84e)